Automattic/jetpack
# Dashboard: ensure that the image CDN does not get enabled by default when using the Site Accelerator toggle while in Offline mode.

#### Changes proposed in this Pull Request:

﻿The Site Accelerator toggle is a custom toggle that works a bit differently from the standard ModuleToggle component. Since it handles 2 modules at once (Asset CDN and Photon), it needs to check if the 2 modules can actually be toggled on before to activate them. Until now, we checked if the Photon module had an override set via a filter, but we did not check if it was rendered unavailable via Offline mode. This changes that.

This was brought up in https://github.com/Automattic/jetpack/pull/20534#issuecomment-892836173

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

* Start with a site in [Offline mode](https://jetpack.com/support/development-mode/)
* Go to Jetpack > Settings > Performance
* The image CDN toggle should be grayed out.
* Toggle the Site Accelerator toggle. The asset CDN option should get turned on, but not the image CDN.
